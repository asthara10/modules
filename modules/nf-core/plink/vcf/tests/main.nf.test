nextflow_process {

    name "Test Process PLINK_VCF"
    script "../main.nf"
    config "./nextflow.config"
    process "PLINK_VCF"

    tag "modules"
    tag "modules_nfcore"
    tag "plink"
    tag "plink/vcf"

    test("test_plink_vcf") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test', single_end:false ], // meta map
              file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true) ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
