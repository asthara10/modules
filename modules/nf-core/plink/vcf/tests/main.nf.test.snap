{
    "versions": {
        "content": [
            [
                "versions.yml:md5,e8954e602ae53e958694bbc72e67abca"
            ]
        ],
        "timestamp": "2023-11-24T14:22:56.78541"
    }
}